var/spack/repos/builtin/packages/openpmd-api/package.py

```diff
@@ -41,7 +41,8 @@ class OpenpmdApi(CMakePackage):
     version("0.12.0", tag="0.12.0-alpha", commit="23be484dd2570b5277779eafcc5f1eb70c6d98f2")
     version("0.11.1", tag="0.11.1-alpha", commit="c40292aafbf564807710424d106304f9670a8304")
 
-    depends_on("cxx", type="build")  # generated
+    depends_on("c", type="build")
+    depends_on("cxx", type="build")
```

Comment:
```suggestion
    depends_on("cxx", type="build")
```
The only immediate language dependency for the openPMD-api is on C++

Reply:
These additional dependencies stem from a branch where we error if a language wrapper is used without declaring the dependency in the package, as explained in https://github.com/spack/spack/pull/47662#issuecomment-2484180106, https://github.com/spack/spack/pull/47662#issuecomment-2484192915, and https://github.com/spack/spack/pull/47662#discussion_r1847331974

Also. checking openPMD CMakeLists I see:

- https://github.com/openPMD/openPMD-api/blob/d1015eefb766aee129283851fdabfb7e8935e698/CMakeLists.txt#L5C1-L5C32 

and [CMake docs](https://cmake.org/cmake/help/latest/command/project.html) says:

> By default C and CXX are enabled if no language options are given. Specify language NONE, or use the LANGUAGES keyword and list no languages, to skip enabling any languages.

You can verify this by using this dummy CMakeLists.txt:
```cmake
cmake_minimum_required(VERSION 3.22.0)

project(openPMD VERSION 0.17.0) # LANGUAGES CXX
```
and running `cmake` over that. If, instead of a comment, `LANGUAGES` was an option given to the project command, I would agree with you :wink:

Reply:
Huh, good point. That line seems to be from 2018 (https://github.com/openPMD/openPMD-api/commit/b17feeb09c0a02f5630b81a45f9fcad5b5057d35) when the project started out, so maybe I should take the chance to change that now. 
In that case, ignore my suggested change, this PR is fine from openPMD's perspective.